LegStamped A_LF
LegStamped B_RF
LegStamped C_RH
LegStamped D_LH
